cms/djangoapps/contentstore/rest_api/v0/views/advanced_settings.py

```diff
@@ -39,9 +40,14 @@ def clean_filter_fields(self):
             apidocs.string_parameter("course_id", apidocs.ParameterLocation.PATH, description="Course ID"),
             apidocs.string_parameter(
                 "filter_fields",
-                apidocs.ParameterLocation.PATH,
+                apidocs.ParameterLocation.QUERY,
                 description="Comma separated list of fields to filter",
             ),
+            apidocs.string_parameter(
```

Comment:
Is this for the purpose of showing and hiding the deprecated fields in advanced settings?

Reply:
This is done to show/hide settings in response based on the feature flag (i.e., enabled or disabled).